### ISSUE TITLE
Cart total just freaked out?

### ISSUE THREAD

**Issue by ismurray** [RaindropAndWee/raindrop-client#15]
looks like this: 
```
Active Shopping Cart:
Nodemon
ID: 5ab799400014728d3681afd7
Price: $1749.99
Description: Legendary Digimon that lives in the depths of the command line.
Remove from Cart
Headphones
ID: 5ab799400014728d3681afd5
Price: $14.99
Description: For your muzaks.
Remove from Cart
Dragon
ID: 5ab799400014728d3681afd3
Price: $999.99
Description: Hide them in your dungeon!
Remove from Cart
Total: 2764.9700000000003
```

**Comment by ismurray** [comment]
Also cart total doesn't update on addItemToCart, only updates on remove item

**Comment by ismurray** [comment]
Closed by commit 541b8a7
